x-pack/plugins/enterprise_search/public/applications/app_search/components/error_connecting/error_connecting.tsx

```diff
@@ -13,14 +13,16 @@ import { ErrorStatePrompt } from '../../../shared/error_state';
 import { SetAppSearchChrome as SetPageChrome } from '../../../shared/kibana_chrome';
 import { SendAppSearchTelemetry as SendTelemetry } from '../../../shared/telemetry';
 
-export const ErrorConnecting: React.FC = () => {
+export const ErrorConnecting: React.FC<{ errorConnectingMessage?: string }> = ({
```

Comment:
Seems a bit redundant to include "error connecting" in the prop name when the component name also does so.